docs/docs/api/promise.on.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+id: promise.on
+title: Promise.on
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.on
+
+```js
+Promise.on(String eventName("created"|"chained"|"fulfilled"|"rejected"|"resolved"|"cancelled"), Function handler, [Boolean simpleEventAPI]) -> undefined
+```
+Subscribes a handler function for an event from promise lifecycle ("created", "chained", "fulfilled", "rejected", "resolved" and "cancelled").
+Event handlers can be of 2 kinds:
+1) Handler receives an event object in given format:
+```js
+function(Object:{guid: Number, childGuid: Number, eventName: String, detail: any, timeStamp: Date, stack: Trace}) -> undefined;
```

Comment:
I think it would be more readable to explain each of these parameters in a separate line. That signature is long and what `stack` means in the context of a `resolved` event isn't very clear here.